vscode-lean4/src/utils/leanInstaller.ts

```diff
@@ -21,6 +21,7 @@ export class LeanInstaller implements Disposable {
     private subscriptions: Disposable[] = [];
     private prompting : boolean = false;
     private defaultToolchain : string;
+    private elanDefaultToolchain : string = '';
```

Comment:
I am a bit confused about the difference between `defaultToolchain` and `elanDefaultToolchain`. Do we need both? If so, could you document this field explaining why?

Reply:
I added comments.